docker/etna-base/entrypoints/build.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
```

Comment:
The previous 'ad-hoc' entrypoints have been scratched in favor of shareable, consistent entrypoints that handle the two main configurations of docker images: "building" via installation and preparation of assets, and "development" configuration such as linking etna and waiting for database readiness before starting.

While most etna applications can easily share logic for these entrypoints, there are some notable exceptions where special logic is required.  To aid in this, both build.sh and development.sh support "hooks" that projects provide.  `magma` and `janus` use these currently.

README.md covering `etna-base` and `entrypoints` coming next.